src/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Contenta Angular",
-  "short_name": "Contenta",
+  "name": "Umami Magazine",
```

Comment:
I personally believe we should mention contenta still somewhere on the website :)